### ISSUE TITLE
Please read once (Important)

### ISSUE THREAD

**Issue by tamal78** [Abhiramborige/CAO_Shortcuts#4]
please don't add any data structures or algorithms or interview question as it may violate rules of hacktoberfest.
u can add notes of that..
but dont do problems and solutions..
ur repo may be excluded from event

**Comment by Abhiramborige** [comment]
Yeah I got it but why did you come forward to contribute code to my repo. If I am bit pointing in nature, then please sorry. I too have to maintain the decorum of the event, so I am also sorry for that. Now, I will unmark your PR from being "hactoberfest" and "hactoberfest-accepted".
